CompilerSource/settings-parse/parse_ide_settings.cpp

```diff
@@ -139,6 +140,9 @@ void parse_ide_settings(const char* eyaml)
   setting::automatic_semicolons   = settree.get("automatic-semicolons").toBool();
   setting::keyword_blacklist = settree.get("keyword-blacklist").toString();
 
+  // The number of compile jobs
+  num_make_jobs = settree.get("jobs").toString();
```

Comment:
Probably worth round-tripping to int to make sure we don't pass anything weird to make.

Reply:
It is defined to be an int, the parsing should fail if it isn't
```cpp
("jobs,j", opt::value<int>()->default_value(1), "The number of compile jobs to run simultaneously")
```

Reply:
This string isn't just built from emake flags; it can be read from elsewhere (e.g. when prepared in LGM), too (and indeed, LGM will not be specifying a value for it, so the YAML parser will come back with the empty string).